operation-common/src/main/java/org/sopt/makers/operation/code/failure/auth/AuthFailureCode.java

```diff
@@ -0,0 +1,27 @@
+package org.sopt.makers.operation.code.failure.auth;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.sopt.makers.operation.code.failure.FailureCode;
+import org.springframework.http.HttpStatus;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@Getter
+@RequiredArgsConstructor
+public enum AuthFailureCode implements FailureCode {
+    // 400
```

Comment:
`P4`

별 건 아니구 `class`와 내부 내용 사이에 개행!!
```suggestion
public enum AuthFailureCode implements FailureCode {

    // 400
```